plugins/outputs/quix/quix.go

```diff
@@ -0,0 +1,130 @@
+// quix.go
+package quix
+
+import (
+	"crypto/sha256"
+	"strings"
+
+	"github.com/IBM/sarama"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+)
+
+// Quix is the main struct for the Quix plugin
+type Quix struct {
+	Brokers        []string `toml:"brokers"`
+	Topic          string   `toml:"topic"`
+	Workspace      string   `toml:"workspace"`
+	AuthToken      string   `toml:"auth_token"`
+	APIURL         string   `toml:"api_url"`
+	TimestampUnits string   `toml:"timestamp_units"`
+
+	producer   sarama.SyncProducer
+	Log        telegraf.Logger
+	serializer serializers.Serializer
+}
+
+// SampleConfig returns a sample configuration for the Quix plugin
+func (q *Quix) SampleConfig() string {
+	return `
+  ## Quix output plugin configuration
+  workspace = "your_workspace"
+  auth_token = "your_auth_token"
+  api_url = "https://portal-api.platform.quix.io"
+  topic = "telegraf_metrics"
+  data_format = "json" 
+  timestamp_units = "1s"
+`
+}
```

Comment:
You need to embed the `sample.conf` file here so we do have a single-source-of-truth!

```suggestion
func (*Quix) SampleConfig() string {
	return sampleConfig
}
```